code/game/objects/structures/catwalk.dm

```diff
@@ -67,4 +66,35 @@
 			//Lattice would delete itself, but let's save ourselves a new obj
 			if(istype(src.loc, /turf/space) || istype(src.loc, /turf/simulated/open))
 				new /obj/structure/lattice/(src.loc)
+			if(plated_tile)
+				new plated_tile.build_type(src.loc)
 			qdel(src)
+		return
+	if(istype(C, /obj/item/stack/tile/mono) && !plated_tile)
+		var/obj/item/stack/tile/floor/ST = C
+		if(!ST.in_use)
+			to_chat(user, "<span class='notice'>Placing tile...</span>")
+			ST.in_use = 1
+			if (!do_after(user, 10))
+				ST.in_use = 0
+				return
+			to_chat(user, "<span class='notice'>You plate the catwalk</span>")
```

Comment:
grammar

Reply:
This is the correct past tense, I will add the dot

Reply:
its present tense and the tense is fine, I just wanted the period. Bay has a tendency to miss things like that.